src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -0,0 +1,1276 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Formats.Asn1;
+using System.Security.Cryptography.Asn1;
+
+namespace System.Security.Cryptography
```

Comment:
You need to pragma suppress the IDE0060 unused parameter errors until the methods have bodies.